packages/components/src/internal/components/domainproperties/ConfirmImportTypes.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
+
+import {ConfirmModal} from "../../..";
+
+interface Props {
+    error: any;
+    show: boolean;
+    onConfirm: () => void;
+    onCancel: () => void;
+    designerType: String;
+}
+
+interface State {
+}
+
+export default class ConfirmImportTypes extends PureComponent<Props, State> {
+    render() {
+        const { error, onConfirm, onCancel, show, designerType } = this.props;
+
+        return (
+            <ConfirmModal
+                show={show}
+                title='Error Importing File'
+                msg={
+                    <>
+                        <div>
+                            <p>
+                                There was an error while trying to import the selected file. <b> Should you choose to continue,
+                                 no importable data will be present in your created {designerType}. </b> You may resolve the problem below by
+                                returning to your Field Editor and rectifying the error.
```

Comment:
Can you sent a screenshot of this dialog to MattY and get his input on wording for this text and wording for the buttons? I think what you have makes sense but might be a bit too technical.

Reply:
Sent, pending response!